pkg/exchange/okex/exchange.go

```diff
@@ -414,3 +417,25 @@ func (e *Exchange) QueryOrderTrades(ctx context.Context, q types.OrderQuery) ([]
 
 	return trades, nil
 }
+
+func (e *Exchange) SupportedInterval() map[types.Interval]int {
+	return okexapi.SupportedIntervals
+}
+
+func (e *Exchange) IsSupportedInterval(interval types.Interval) bool {
+	_, ok := okexapi.SupportedIntervals[interval]
+	return ok
+}
+
+func (e *Exchange) GetAllFeeRates(ctx context.Context) (okexapi.FeeRateList, error) {
```

Comment:
Why you need GetAllFeeRates?

Reply:
Bybit uses it because the stream event does not return the fee and the fee currency.

Reply:
ok, I will remove it, I found out 'order' websocket would return fee, so I don't need GetAllFeeRates
https://www.okx.com/docs-v5/en/?shell#order-book-trading-trade-ws-order-channel